com.logseq.Logseq.json

```diff
@@ -21,7 +21,8 @@
         "--socket=x11",
         "--socket=pulseaudio",
         "--share=network",
-        "--filesystem=home"
+        "--filesystem=home",
+        "--persist=.logseq"
```

Comment:
this shouldn't be needed since it has `home` permissions. Are you sure it fixes it?

Reply:
oops, yeah, that was a bit overzealous of me :see_no_evil:
background: I personally override flatpaks to disable `home` permissions and only share what is relevant to the app. So I just merrily did my usual routine without thinking it through. :man_facepalming: 

Sorry for the noise!